strato_query/__tests__/test_api_query.py

```diff
@@ -686,3 +686,49 @@ def test_pure_shape_union_query(self):
                 ]))
         assert 'geometry' in df['FEATURES'][0][0]
         assert len(df['FEATURES'][0]) == 2
+
+    def test_miles_distance_query(self):
+        df = self.submit_query(
+            query_params=APIMilesDistanceQueryParams(
+                start_latitude=41.82937349570897,
+                start_longitude=-71.41338050365448,
+                end_latitude=42.35339843570063,
+                end_longitude=-71.06788516044617))
+        assert 'DISTANCE' in df
+        assert df['DISTANCE'][0] == 40.3163963879, df['DISTANCE'][0]
+
+    def test_driving_distance_query(self):
+        df = self.submit_query(
+            query_params=APIDrivingDistanceQueryParams(
+                start_latitude=41.82937349570897,
+                start_longitude=-71.41338050365448,
+                end_latitude=42.35339843570063,
+                end_longitude=-71.06788516044617))
+        assert 'DISTANCE' in df
+        assert df['DISTANCE'][0] == 44.96875, df['DISTANCE'][0]
+        assert 'TIME' in df
+        assert df['TIME'][0] == 60.0833333333, df['TIME'][0]
+
+        df = self.submit_query(
+            query_params=APIDrivingDistanceQueryParams(
+                traffic=True,
+                start_latitude=41.82937349570897,
+                start_longitude=-71.41338050365448,
+                end_latitude=42.35339843570063,
+                end_longitude=-71.06788516044617))
+        assert 'DISTANCE' in df
+        assert df['DISTANCE'][0] == 44.96875, df['DISTANCE'][0]
+        assert 'TIME' in df
+        assert df['TIME'][0] == 55.7333333333, df['TIME'][0]
+
+    def test_walking_distance_query(self):
+        df = self.submit_query(
+            query_params=APIWalkingDistanceQueryParams(
+                start_latitude=41.82937349570897,
+                start_longitude=-71.41338050365448,
+                end_latitude=41.8222540665299,
+                end_longitude=-71.41181409358978))
+        assert 'DISTANCE' in df
+        assert df['DISTANCE'][0] == 0.5761363636, df['DISTANCE'][0]
+        assert 'TIME' in df
+        assert df['TIME'][0] == 17.4333333333, df['TIME'][0]
```

Comment:
Need to do almost equals I think for these

Reply:
Darn, you're right. For some reason I thought that wasn't available in this repo. I'll change my rounding to that.